carbon-design-system/carbon-addons-iot-react
# feat(repo):  move text strings into i18n prop

Closes #1811 

**Summary**

- moves all "non-falsy" strings into an `i18n` prop to bring consistency to all components

**Change List (commits, features, bugs, etc)**

- deprecates the existing prop
- sets the existing prop to `undefined`
- add the prop to an `i18n` prop

**Acceptance Test (how to verify the PR)**

- check that all non-falsey strings that should be translated are in an i18n and their existing prop is deprecated

**Regression Test (how to make sure this PR doesn't break old functionality)**

- confirm no snapshot changes
- confirm no warning in test output




**Things to look for during review**
